web/src/buddy_list.js

```diff
@@ -304,8 +306,8 @@ export class BuddyList extends BuddyListConf {
         padded_widget.update_padding({
             shown_rows: this.render_count,
             total_rows: this.keys.length,
-            content_sel: this.container_sel,
-            padding_sel: this.padding_sel,
+            content_sel: this.container_selector,
+            padding_selector: this.padding_selector,
```

Comment:
Sorry, I missed two references that I for some reason thought weren't connected but definitely are: #27808